src/components/flow/actions/changegroups/addgroups/AddGroupsForm.tsx

```diff
@@ -85,7 +85,6 @@ export default class AddGroupsForm extends React.Component<ActionFormProps, Chan
           searchable={true}
           shouldExclude={excludeDynamicGroups}
           placeholder={i18n.t('select_groups', 'Select Groups')}
-          expressions={true}
```

Comment:
Let's wait for the definition on https://app.asana.com/0/1159876841343611/1199948461614664/f before pushing that to dev.ccl